sherpa/astro/xspec/src/_xspec.cc

```diff
@@ -1405,6 +1423,15 @@ static PyMethodDef XSpecMethods[] = {
   XSPECMODELFCT_C(C_log10con, 1),
 #endif
 
+#ifdef XSPEC_12_12_0
+  XSPECMODELFCT_C_NORM( xsgrbjet, 14 ),  // follow xsgrbcomp and drop the leading c_
+  XSPECMODELFCT_C_NORM( C_vvwDem, 37 ),
+  XSPECMODELFCT_C_NORM( C_vwDem, 21 ),
+  XSPECMODELFCT_C_NORM( C_wDem, 8 ),
+
+  XSPECMODELFCT(zxipab, 5),
```

Comment:
It seems that 'xs' is missing from zxipab model name also here

Reply:
No, this is the correct name for this part of the code.